### PR TITLE
165 student provides financial information travel costs

### PR DESCRIPTION
**Pull request by 2011sandeepraj**
image

Expected functionality:
If No is selected to "living situation" question, the travel question is shown. If a yes is selected, the section asking for cost of one return trip home is shown.

Reviewers please note a couple of points:
1. questions-financial.ts: I want to have all the questions here. the file FinancialInfo.vue is getting big and hard to navigate. If the devs agree, I can move all questions for questions-financial into this file.
2. RadioButtonList.ts: This now has a default yes/no array. So all places where "options:yesNoOptions" was found to be used has been removed

### CODE REVIEW

File: sources/packages/web/src/helpers/questions-financial.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+export const questionsFin = {
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/helpers/questions-financial.ts:1]: I do not see a problem moving the questions, but I would prefer that they are moved inside a specific folder for fa-application, and please move all others also.

File: sources/packages/web/src/views/FinancialInfo.vue [hunk 2]
```diff
@@ -49,6 +49,48 @@
...
+
+              <ContentGroup
+                v-if="financialInfoState.travelCostsValue === 'yes'"
+              >
+                <Section>
+                  <Question :text="questionsFin.returnTripCosts.question">
+                    <InputNumber
+                      class="p-m-2"
+                      mode="currency"
+                      currency="CAD"
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/views/FinancialInfo.vue:82]: Please take a look to ensure that the currency symbol is "$". It seems to be the requirement.

File: sources/packages/web/src/views/FinancialInfo.vue [hunk 3]
```diff
@@ -79,23 +130,21 @@ const questions = {
...
     HorizontalSeparator,
     StringToHtmlParagraphs,
+    RadioButtonList,
     ReadMoreContainer,
   },
   setup() {
     const financialInfoState = reactive({} as FinancialInfoState);
     return {
       questions,
+      questionsFin,
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/views/FinancialInfo.vue:147]: Please move all the questions to the new source.

**Review by andrewsignori-aot — CHANGES_REQUESTED** [review]
Please check the comments.

File: sources/packages/web/src/helpers/questions-financial.ts (hunk 1 above)
Comment (fwpushan) [reply sources/packages/web/src/helpers/questions-financial.ts:1]: @andrewsignori-aot @2011sandeepraj 
1. I've created one [technical debt](https://app.zenhub.com/workspaces/student-information-management-system-5fce9df5aa1b45000e937014/issues/bcgov/sims/188) to manage static texts of the application and questionnaire.
2.  We have one placeholder for design challenge on questionnaire model, that may change whole section.  So, I will recommend, keep this portion in view/component.

**Review by fwpushan — APPROVED** [review]
LGTM to me

File: sources/.docker/postgres/create_db.sh [hunk 4]
```diff
@@ -18,4 +18,4 @@ if [ -n "$POSTGRES_MULTIPLE_DATABASES" ]; then
     create_user_and_database $db
   done
   echo "Multiple databases created"
-fi
+fi
```
Comment (andrewsignori-aot) [inline sources/.docker/postgres/create_db.sh:21]: Please avoid adding this to the commit if it is not necessary.

File: sources/packages/web/src/validators/ValidatorConfigUI.ts [hunk 5]
```diff
@@ -6,5 +6,7 @@ export default function configValidationRules() {
   // A list of available rules to be used could be found on
   // https://vee-validate.logaretm.com/v4/guide/global-validators#available-rules
   defineRule("required", required);
+  console.log(`In validator Config UI `);
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/validators/ValidatorConfigUI.ts:9]: Please remove commented lines.

**Review by andrewsignori-aot — COMMENTED** [review]
Nice work moving the questions to the constants.
Please take a look at the minor comments.

**Review by andrewsignori-aot — APPROVED** [review]
After fixing the minor comments it should be good to go 👍

File: sources/.docker/postgres/create_db.sh (hunk 4 above)
Comment (2011sandeepraj) [reply sources/.docker/postgres/create_db.sh:21]: Unable to

**Pull request merged**